tests/e2e/tests/content-releases/releases-page.spec.ts

```diff
@@ -111,9 +119,8 @@ describeOnCondition(edition === 'EE')('Releases page', () => {
 
       // Navigate to the content manager
       await page.getByRole('link', { name: 'Open the Content Manager' }).click();
-      await expect(page).toHaveTitle('Content Manager');
+      await page.waitForURL('/admin/content-manager/collection-types/*');
```

Comment:
I think we can clickAndWait again, we just need to make sure we arrived on the page, we don't need to explicitly check that the CM URL is the right one